ccd-definition/CaseEvent/CareSupervision/MultiState-nonprod.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "draftSDO",
+    "Name": "Add the gatekeeping order",
+    "Description": "Add the gatekeeping order",
+    "DisplayOrder": 9,
+    "PreConditionState(s)": "Gatekeeping;PREPARE_FOR_HEARING",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/draft-standard-directions/about-to-start",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/draft-standard-directions/about-to-submit",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/draft-standard-directions/submitted",
+    "EndButtonLabel": "Save and continue",
+    "ShowEventNotes": "Y",
```

Comment:
AC 4 requested that it have the CYA, event notes are in the current flow so I kept them as otherwise it would be altering the other flows. If we decide we don't want them in the end I think it can be removed in the rework rather than do them here